.travis.yml

```diff
@@ -53,6 +53,8 @@ addons:
   apt:
     packages:
     - nsis
+    - imagemagick
+    - inkscape
```

Comment:
a test if `inkscape` is installed should be added in `script/makeinstaller.sh`

https://github.com/streamlink/streamlink/blob/2c6e4924b4fc1a567dbdffdfb2df71f6aa7add66/script/makeinstaller.sh#L6-L7


```sh
command -v inkscape > /dev/null 2>&1 || { echo >&2 "inkscape is required to build the installer. Aborting."; exit 1; }
```

Reply:
Done, I also added one for imagemagick. Good idea.